ui/src/components/Issuer/IssueAsset.tsx

```diff
@@ -84,14 +114,12 @@ const IssueAsset = () => {
                 offLabel='Private'
                 offInfo='Only a set of parties will be aware of this token.'
                 onClick={val => setIsPublic(val)}/>
-            <Form.TextArea
+            <Form.Select
+                multiple
```

Comment:
Have you tried using a ContractSelect with `allowAdditions`? This select won't work for a user who wants to add a private party ID as an observer.

You'll have to add and pass through the `multiple` prop to it though.

Reply:
as for this - I just went down a rabbit hole adding `multiple` to the contract selector, and u cant actually add an addition _and_ select multiple at once. So Im thinking we hold off for now and address that with potentially another input field

Reply:
ah I see, no worries then. thanks for investigating that!